### PR TITLE
Change mme-exporter to have TAC and edgename from json file

### PR DESCRIPTION
**Pull request by woojoong88**
Change mme-exporter to have TAC and edgename from json file

### CODE REVIEW

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

File: pyserver/monitor_client.py [hunk 1]
```diff
@@ -1,23 +1,25 @@
-
-from flask import Flask, render_template, request, jsonify, Response
+# Copyright (c) 2019 Open Networking Foundation
```
Comment (ghost) [inline pyserver/monitor_client.py]: please use short header

Copyright 2019-present Open Networking Foundation
SPDX-License-Identifier: Apache-2.0

Comment (woojoong88) [reply pyserver/monitor_client.py]: Got it

Comment (woojoong88) [reply pyserver/monitor_client.py]: Done.

**Comment by thakurajayL** [comment]
Will exporter part of same mme image or we still have 2 separate images ?

**Comment by woojoong88** [comment]
2 separate images.

**Review by thakurajayL — APPROVED** [review]

**Pull request merged**